src/common/header/HeaderNav.jsx

```diff
@@ -30,6 +30,55 @@ const HeaderNav = ({ showBrowse }) => {
 
   const modalClose = () => setShowShareModal(!showShareModal);
 
+  const links = [
```

Comment:
rename to `NavLinks`
```suggestion
  const NavLinks = [
```